ngmix/tests/test_flags.py

```diff
@@ -11,8 +11,6 @@
 def test_get_flags_str(dtype):
     """Test that get_flags_str works for different integer types."""
     # Set a high-enough bit in addition to the lower bit to ensure things work.
-    val = np.array(LOW_DET, dtype=dtype)
-    if val.itemsize > 4 and dtype not in (np.uint64, ):
-        val |= 2**45
+    val = np.array(LOW_DET | 2**45).astype(dtype)
```

Comment:
Alright, this looks much cleaned than the test with all the conditions.

Reply:
Yeah it just leaves the question of how numpy handles truncating things. That's ok I guess but maybe a comment would be good.